src/status_im/subs/wallet/wallet.cljs

```diff
@@ -261,22 +263,22 @@
  :wallet/settings-keypairs-accounts
  :<- [:wallet/keypairs]
  (fn [keypairs [_ format-options]]
-   (let [grouped-keypairs     (group-by :lowest-operability keypairs)
+   (let [grouped-keypairs     (->> keypairs
+                                   vals
+                                   (group-by :lowest-operability))
          operable-keypair-ids (->> (concat (:fully grouped-keypairs)
                                            (:partially grouped-keypairs))
-                                   (map :key-uid)
-                                   (into #{}))
-         missing-keypair-ids  (->> (map :key-uid (:no grouped-keypairs))
-                                   (into #{}))]
-     {:operable (->> keypairs
-                     (filter #(contains? operable-keypair-ids (:key-uid %)))
+                                   (mapv :key-uid))
+         missing-keypair-ids  (mapv :key-uid (:no grouped-keypairs))]
+     {:operable (->> (select-keys keypairs operable-keypair-ids)
+                     vals
                      (map (fn [{:keys [accounts name type key-uid]}]
                             {:type     (keyword type)
```

Comment:
Not a change made in this PR, but this piece of code is very error-prone.

I'd suggest to rename the destructuring of `type` to `key-uid-type` (or something that fits better), because we are applying `keyword` to it.

Similar for `name`